keras_resnet/models/_1d.py

```diff
@@ -129,20 +150,20 @@ class ResNet1D18(ResNet1D):
 
         >>> import keras_resnet.models
 
-        >>> shape, classes = (224, 224, 3), 1000
+        >>> shape, classes = (224, 3), 1000
```

Comment:
Running this with the fixed file still gives this error:
`ValueError: Input 0 of layer conv1 is incompatible with the layer: : expected min_ndim=4, found ndim=3. Full shape received: (None, 224, 3)`